### PR TITLE
BABEL-3781 Bigint/Int/Smallint/Tinyint Radians Function

### PR DESCRIPTION
**Pull request by pratikzode**
This Jira is a part of BABEL-736.
At first in Babel, the Transact-SQL functions radians always return float instead of returning the same type in SQL server. In this Jira we have added Bigint, Int, Smallint and Tinyint to radians function. Have added the test case to show Bigint, Int, Smallint and Tinyint to radians. Now this 3 is giving same output as SQL server.

Task: BABEL-3781
Signed-off-by: pratikzode <pratik.zode@gmail.com>

### CODE REVIEW

File: contrib/babelfishpg_tsql/runtime/functions.c [hunk 1]
```diff
@@ -1254,3 +1255,45 @@ smallint_degrees(PG_FUNCTION_ARGS)
...
+{
+	int64    arg1 = PG_GETARG_INT64(0);
+	float8  result;
+
+	result = DatumGetFloat8(DirectFunctionCall1(radians, Float8GetDatum((float8) arg1)));
+
+	/* skip range check, since it cannot overflow int64 */
+
+	PG_RETURN_INT64((int64)result);
+
```
Comment (macwoody) [inline contrib/babelfishpg_tsql/runtime/functions.c]: plz remove this new line

File: contrib/babelfishpg_tsql/runtime/functions.c [hunk 2]
```diff
@@ -1254,3 +1255,45 @@ smallint_degrees(PG_FUNCTION_ARGS)
...
+{
+	int32    arg1 = PG_GETARG_INT32(0);
+	float8  result;
+
+	result = DatumGetFloat8(DirectFunctionCall1(radians, Float8GetDatum((float8) arg1)));
+
+	/* skip range check, since it cannot overflow int32 */
+
+	PG_RETURN_INT32((int32)result);
+
```
Comment (macwoody) [inline contrib/babelfishpg_tsql/runtime/functions.c]: same here

File: contrib/babelfishpg_tsql/runtime/functions.c [hunk 3]
```diff
@@ -1254,3 +1255,45 @@ smallint_degrees(PG_FUNCTION_ARGS)
...
+smallint_radians(PG_FUNCTION_ARGS)
+{
+	int16    arg1 = PG_GETARG_INT16(0);
+	float8  result;
+
+	result = DatumGetFloat8(DirectFunctionCall1(radians, Float8GetDatum((float8) arg1)));
+
+	/* skip range check, since it cannot overflow int32 */
+
+	PG_RETURN_INT16((int16)result);
```
Comment (macwoody) [inline contrib/babelfishpg_tsql/runtime/functions.c]: you are returning INT16

File: contrib/babelfishpg_tsql/runtime/functions.c [hunk 4]
```diff
@@ -1254,3 +1255,45 @@ smallint_degrees(PG_FUNCTION_ARGS)
...
+
```
Comment (macwoody) [inline contrib/babelfishpg_tsql/runtime/functions.c]: same here

File: test/JDBC/expected/BABEL-3781-vu-prepare.out [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+		e tinyint,
+);
+GO
+
+
+INSERT INTO BABEL_3781_vu_prepare_t_1 VALUES (NULL, NULL, NULL, NULL, NULL);
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO BABEL_3781_vu_prepare_t_1 VALUES (2147483647, NULL, 9223372036854775807, 32767, 255);
```
Comment (macwoody) [inline test/JDBC/expected/BABEL-3781-vu-prepare.out]: could you please let me know what is the intension of the NULL for column b here?

File: test/JDBC/expected/BABEL-3781-vu-prepare.out [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+INSERT INTO BABEL_3781_vu_prepare_t_1 VALUES (-2147483648, 200.1,-9223372036854775808, -32768, 0);
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO BABEL_3781_vu_prepare_t_1 VALUES (101.23, 20.1, 97777.32, NULL, 120.32);
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO BABEL_3781_vu_prepare_t_1 VALUES (243.2, 451.1, NULL, 904, 80);
```
Comment (macwoody) [inline test/JDBC/expected/BABEL-3781-vu-prepare.out]: same here for column c

File: test/JDBC/expected/BABEL-3781-vu-prepare.out [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+
+INSERT INTO BABEL_3781_vu_prepare_t_1 VALUES (2147483647, NULL, 9223372036854775807, 32767, 255);
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO BABEL_3781_vu_prepare_t_1 VALUES (-2147483648, 200.1,-9223372036854775808, -32768, 0);
+GO
+~~ROW COUNT: 1~~
+
+INSERT INTO BABEL_3781_vu_prepare_t_1 VALUES (101.23, 20.1, 97777.32, NULL, 120.32);
```
Comment (macwoody) [inline test/JDBC/expected/BABEL-3781-vu-prepare.out]: same here for column d

Comment (pratikzode) [reply test/JDBC/expected/BABEL-3781-vu-prepare.out]: Made the Changes

**Review by xhfanhe — APPROVED** [review]

File: test/JDBC/upgrade/latest/schedule [hunk 8]
```diff
@@ -102,6 +102,7 @@ BABEL-2917
 BABEL-2955
 BABEL-3358
 BABEL-3747
+BABEL-3781
```
Comment (surendravishnoi) [inline test/JDBC/upgrade/latest/schedule:105]: We should use PR template for commit message. Also, latest schedule file is for Major version upgrade testing. To test it for minor version upgrade, add it to 14.6 schedule file

**Pull request merged**